Makefile

```diff
@@ -14,9 +14,16 @@ $(TARG): $(GOFILES)
 $(TARG).1: $(TARG)
 	./$(TARG) --help-man > $@
 
-install: $(TARG) $(TARG).1
+copy: 
 	install -m755 $(TARG) $(bindir)/$(TARG)
+	install -d -m644 $(man1dir)
 	install -m644 $(TARG).1 $(man1dir)/$(TARG).1
 
+install: copy clean
```

Comment:
The problem is that if you run go with Sudo, it will throw an error, so my suggestion is based on not having to call go together with a command that usually requires sudo, ever.  Do you think its better that people just get the GO error thrown mentioned in issue:#27 ?